aigit/main.py

```diff
@@ -24,13 +24,16 @@ def main():
         return
 
     # Check git status and get diffs
-    diffs = check_git_status()
-    if not diffs:
+    status = check_git_status()
+    if not status:
         print("No changes detected.")
         return
 
+    diffs = status["changed_files"]
+    new_files = status["new_files"]
+
     # Generate commit message using OpenAI
-    commit_message = generate_commit_message(diffs, api_key)
+    commit_message = generate_commit_message(diffs, new_files, api_key)
```

Comment:
@mentantbot You are making a mistake here. The `generate_commit_message` method only takes 2 arguments as defined in `aigit/openai_interface.py` You also need to update the method there so that it takes into consideration new files in the generated commit. Update it.